README.md

```diff
@@ -84,136 +85,84 @@ An agent component for Java system.
 > You can use `git config --global core.autocrlf false` to modify `core.autocrlf`.
 
-### 1. Get Configuration file
+#### Get Configuration file
 Extracting the default configuration file.
 ```
+$ cd $EASE_AGENT_PATH
 $ jar xf easeagent.jar agent.properties easeagent-log4j2.xml
 ```
 
-### 2. Modify Configuration
-* Modify service name, default configuration is `demo-service`.
-```
-name=[app-name]
-```
-* Modify kafka server config, default configuration is `127.0.0.1:9092`.
-Both `tracing` data and `metric` data will be send to kafka server by default configuration.
-```
-observability.outputServer.bootstrapServer = [ip:port]
-```
-
-* Modify output configuration, if you want to watch log information in console.
-```
-# metric output
-observability.metrics.[xxxx].appendType=console
-
-# tracings output to console
-observability.tracings.output.target=system
-observability.tracings.output.enabled=false
-```
-If you want to watch tracing information only in console, comment out the outputServer configuration and set target to 'system':
-```
-# observability.outputServer.bootstrapServer=127.0.0.1:9092
-# observability.outputServer.timeout=10000
-# observability.outputServer.enabled=true
-
-observability.tracings.output.target=system
-
-```
-
-* Sending tracing data to zipkin server
-```
-# [zipkin]: send data to zipkin server
-# [system]: send data to kafka
-observability.tracings.output.target=zipkin
-observability.tracings.output.target.zipkinUrl=http://localhost:9411/api/v2/spans
-```
+By default, there is an agent.properties configuration , which is configured to print all information to the console.
 
-### 3. Building the demo application and simple plugin.
-```
-$ git clone https://github.com/megaease/easeagent-test-demo.git
-$ cd spring-web
-$ mvn clean package
+### Monitor Spring Petclinic
 
-$ cd simple-plugin
-$ mvn clean package
-```
-There is an agent.properties configuration file in the demo directory, which is configured to print all information to the console.
-If you want to print all information to console, then you can use this configuration file.
+Download and init project then start it. You need git and docker:
 
-### 4. Copy simple plugin to `plugins` directory
-There is a [simple plugin](https://github.com/megaease/easeagent-test-demo/tree/master/simple-plugin) which only for demonstrating the use of the plugin. 
-The compiled simple plugin JAR package should be copy to the `plugins` directory located in `EASE_AGENT_PATH`, and if the directory is not exsist, it need to be created.
+[Project Details](https://github.com/megaease/easeagent-spring-petclinic)
 
 ```
-$ export EASE_AGENT_PATH=[Replace with agent path]
-cp target/simple-plugin-1.0.jar $EASE_AGENT_PATH/plugins/
-
+$ git clone https://github.com/megaease/easeagent-spring-petclinic.git
+$ cd easeagent-spring-petclinic
+$ git checkout v2.1.0
+$ mkdir easeagent/downloaded && cp $EASE_AGENT_PATH/easeagent.jar  easeagent/downloaded/easeagent-v2.1.0.jar
```

Comment:
1. The downloaded directory can be created automatically.
2. easeagent.jar can be downloaded from the Github release page 
Two steps can be omitted.

Reply:
> 1. The downloaded directory can be created automatically.
> 2. easeagent.jar can be downloaded from the Github release page
>    Two steps can be omitted.

modified